extensions/typescript/src/features/referenceProvider.ts

```diff
@@ -14,7 +14,7 @@ export default class TypeScriptReferenceSupport implements ReferenceProvider {
 	public constructor(
 		private client: ITypescriptServiceClient) { }
 
-	public provideReferences(document: TextDocument, position: Position, options: { includeDeclaration: boolean }, token: CancellationToken): Promise<Location[]> {
+	public provideReferences(document: TextDocument, position: Position, options: { includeDeclaration: boolean }, _: (value: Location) => void, token: CancellationToken): Promise<Location[]> {
```

Comment:
TSLint/tsc shouldn't complain about that. I wouldn't use this pattern of using an underscore in TS